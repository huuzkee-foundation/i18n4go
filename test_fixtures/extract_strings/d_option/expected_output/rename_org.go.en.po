# filename: d_option/input_files/org/rename_org.go, offset: 806, line: 30, column: 16
msgid "rename-org"
msgstr "rename-org"

# filename: d_option/input_files/org/rename_org.go, offset: 835, line: 31, column: 16
msgid "Rename an org"
msgstr "Rename an org"

# filename: d_option/input_files/org/rename_org.go, offset: 867, line: 32, column: 16
msgid "CF_NAME rename-org ORG NEW_ORG"
msgstr "CF_NAME rename-org ORG NEW_ORG"

# filename: d_option/input_files/org/rename_org.go, offset: 1094, line: 38, column: 20
msgid "Incorrect Usage"
msgstr "Incorrect Usage"

# filename: d_option/input_files/org/rename_org.go, offset: 1452, line: 54, column: 13
msgid "Renaming org %s to %s as %s..."
msgstr "Renaming org %s to %s as %s..."

