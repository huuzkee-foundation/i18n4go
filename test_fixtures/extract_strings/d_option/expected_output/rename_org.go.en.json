[{"id":"rename-org","translation":"rename-org"},{"id":"Rename an org","translation":"Rename an org"},{"id":"CF_NAME rename-org ORG NEW_ORG","translation":"CF_NAME rename-org ORG NEW_ORG"},{"id":"Incorrect Usage","translation":"Incorrect Usage"},{"id":"Renaming org %s to %s as %s...","translation":"Renaming org %s to %s as %s..."}]
